src/rcx/src/gs.cpp

```diff
@@ -145,6 +148,10 @@ void gs::setSize(const int plane,
     exit(-1);
   }
 
+  if (plc.plane != nullptr) {
+    free(plc.plane);
+  }
```

Comment:
No need to check for null before free (again)

Reply:
Indeed